tools/java_api/src/main/java/com/kuzudb/Connection.java

```diff
@@ -31,19 +31,19 @@ private void checkNotDestroyed() throws ObjectRefDestroyedException {
     }
 
     /**
-    * Finalize.
+    * Close.
```

Comment:
The description here should be more descriptive.

Reply:
Changed, please let me know if the new description is more fitting 👍